test/musica/test_musica_api.F90

```diff
@@ -533,7 +585,9 @@ subroutine test_terminator()
                           geopotential_height_wrt_surface_at_interface, surface_geopotential,           &
                           surface_temperature, surface_albedo, num_photolysis_wavelength_grid_sections, &
                           flux_data_photolysis_wavelength_interfaces, extraterrestrial_flux,            &
-                          standard_gravitational_acceleration, cloud_area_fraction, air_pressure_thickness, errmsg, errcode )
+                          standard_gravitational_acceleration, cloud_area_fraction, air_pressure_thickness, latitude, longitude, earth_eccentricity, &
+                          earth_obliquity, perihelion_longitude, moving_vernal_equinox_longitude,       &
```

Comment:
The same request for splitting the arguments into a couple of lines.

Reply:
done!